src/sass/pages/_list-category.scss

```
@@ -0,0 +1,59 @@
+.list-category {
+  .main-heading {
+    @include font($fs: $fs-sizable, $textTrans: uppercase);
+    margin-bottom: 2rem;
+    &::after {
+      background-image: url("/assets/images/v-title.png");
+      top: 2rem;
+    }
+  }
+  .sidebar {
+    &__item {
+      @include font($fs: $fs-light);
+      @include mg($y: 0.5rem);
+      &:hover {
+        cursor: pointer;
+        color: $orange;
+        list-style-type: disc;
+        margin-left: 1rem;
+      }
+    }
+  }
+  .text-compare {
+    @include font($fs: $fs-light);
+  }
+  .tag__list {
```

```suggestion
  }
  
  .text-compare {
    @include font($fs: $fs-light);
  }
  
  .tag__list {
```
những chỗ khác tương tự nhé